bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -679,10 +716,6 @@ private async Task<(string, string, string)> GetLoggedOutDataAsync(string logout
             return (logoutId, logout?.PostLogoutRedirectUri, externalAuthenticationScheme);
         }
 
-        public bool IsNativeClient(IdentityServer4.Models.AuthorizationRequest context)
```

Comment:
Overall, especially for more complex changes, it's better *not* to move around methods, code or other things that don't belong to the specific change you're making, that way our blame and commit history is cleaner. Broader refactors have a place for restructuring things, but not for unrelated PRs. Please revert these reordering changes along with the `#region` updates.

Reply:
I have always employed the boy scout rule of "leave the place better than you found it" as a matter of practice for development, and this was an effort to clean up the code, but I will revert these changes thanks for the feedback.